packages/realm-server/package.json

```diff
@@ -94,7 +94,8 @@
     "lint:js": "eslint . --cache",
     "lint:js:fix": "eslint . --fix",
     "lint:glint": "glint",
-    "full-reset": "./scripts/full-reset.sh"
+    "full-reset": "./scripts/full-reset.sh",
+    "sync-stripe-products": "NODE_NO_WARNINGS=1 PGDATABASE=boxel PGPORT=5435 STRIPE_API_KEY=... ts-node --transpileOnly scripts/sync-stripe-products.ts"
```

Comment:
I'd remove `STRIPE_API_KEY` from here since this is a dynamic value. This should be used like so: `STRIPE_API_KEY=... pnpm sync-stripe-products`